app/src/androidMain/kotlin/de/westnordost/streetcomplete/quests/parking_fee/AddMotorcycleParkingFee.kt

```diff
@@ -0,0 +1,36 @@
+package de.westnordost.streetcomplete.quests.parking_fee
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.geometry.ElementGeometry
+import de.westnordost.streetcomplete.data.osm.osmquests.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.quest.AndroidQuest
+import de.westnordost.streetcomplete.data.user.achievements.EditTypeAchievement.CAR
+import de.westnordost.streetcomplete.osm.Tags
+import de.westnordost.streetcomplete.osm.fee.applyTo
+import de.westnordost.streetcomplete.osm.maxstay.applyTo
+
+class AddMotorcycleParkingFee : OsmFilterQuestType<ParkingFeeAnswer>(), AndroidQuest {
+
+    // element selection logic by @DerDings in #2507
+    override val elementFilter = """
+        nodes, ways, relations with amenity = motorcycle_parking
+        and access ~ yes|customers|public
+        and (
+            !fee and !fee:conditional
+            or fee older today -8 years
+        )
+    """
+    override val changesetComment = "Specify motorcycle parking fees"
+    override val wikiLink = "Tag:amenity=motorcycle_parking"
+    override val icon = R.drawable.ic_quest_motorcycle_parking_fee
+    override val achievements = listOf(CAR)
+
+    override fun getTitle(tags: Map<String, String>) = R.string.quest_motorcycle_parking_fee_title
```

Comment:
I think we can use simply `quest_parking_fee_title` here instead of adding a new string, because

1. the type of object is mentioned below the question
2. the icon already shows a motorcycle
3. it's shorter
4. it's less effort for translators